### PR TITLE
Repo reorganisation

### LINKED ISSUE #52: Repo reorganisation
An issue to document and track the impending repo reorganisation. 

All repos under the me-box organisation are likely to be renamed or archived starting on the 13th July 2017 

### Platform
- [x] databox-app-server --> platform-app-server
- [x] databox-sdk --> plaform-sdk 

### Core
- [x] databox-cm --> core-container-manager 
- [x] databox-logstore --> core-syslog 
- [x] databox-arbiter --> core-arbiter 
- [x] oauth-intermediary --> core-oauth-intermediary
- [x] databox-export-service --> core-export-service 

### Store
- [x] databox-store-blob-mongo --> store-json
- [x] databox-store-iot --> store-timeseries 
- [x] databox-store-hls-video --> store-video

### Lib
- [x] python-databox --> lib-python-databox
- [x] node-databox --> lib-node-databox (Fix npm repo too (does it will have databox-prefix?))

### Source
- [x]  databox-datasource-mock --> source-sdk-mock
- [x]  databox-source-mobile --> source-sensingkit

### Driver 
- [x] databox-driver-mobile --> driver-sensingkit
- [x] databox-driver-google-takeout --> driver-google-takeout
- [x] databox-driver-phillipshue --> driver-phillips-hue
- [x] databox-driver-twitter-stream --> driver-twitter
- [x] databox-os-monitor-driver --> driver-os-monitor

### App
- [x] databox-app-light-graph --> app-light-graph
- [x] databox-app-twitter-sentiment --> app-twitter-sentiment
- [x] Occupancy-as-a-Service --> app-oaas

## To Merge

Merge example apps and drivers into  /samples in the language specific lib 
 
- [x] lib-node  + databox-driver-template-node + databox-app-template-node + databox-app-sensordemo-node + databox-driver-sensordemo-node
- [x] lib-python + databox-driver-template-python

## To Archive
- [x] databox-local-registry
- [x] databox-netdev
- [x] databox-store-mock
- [x] ready-steady-cook
- [x]  databox-vagrant   
- [x] databox-store-blob (Remember to update manifests to use store-json)
- [x] databox-app-mock
- [x] databox-export-destination
- [x] databox-store-passthrough
- [x] databox-driver-template-node 
- [x] databox-app-template-node 
- [x] databox-app-sensordemo-node 
- [x] databox-driver-sensordemo-node
- [x] databox-app-template-java

### PR DESCRIPTION
**Pull request by Toshbrown**
All repose renamed and code update to reflect the changes.

Fixes #52

### CODE REVIEW

File: src/seedManifests.js [hunk 1]
```diff
@@ -28,11 +29,13 @@ const wait = ()=> {
 					console.log("[seeding manifest] get app store root to log in", error);
 					return Promise.reject();
 				}
-				proms = Config.localAppStoreSeedManifests.map((url)=>{
+				proms = Config.localAppStoreSeedManifests.map((name)=>{
 					return new Promise(function(resolve, reject) {
-						req.get(url,(error,response,body)=>{
+						
+						let path = "./"+name+"/databox-manifest.json";
```
Comment (mor1) [inline src/seedManifests.js]: `databox-manifest.json` still? (rather than `manifest.json`?)

File: .gitignore [hunk 2]
```diff
@@ -80,3 +80,29 @@ databox-export-service/
 databox-app-twitter-sentiment/
 
 databox-driver-twitter-stream/
+
+core-arbiter/
+
+core-container-manager/
+
+core-export-service/
```
Comment (mor1) [inline .gitignore:88]: If I were to start getting really picky about naming, perhaps `core-exporter` would fit better with the other core libs? (But really not important or even convincing :)

File: docker-compose-dev-local-images.yaml [hunk 3]
```diff
@@ -3,27 +3,52 @@ version: '3'
 #Build only (put your images in here)
 services:
 #An exmaple of how to add your code to databox
-  databox-os-monitor-driver:
+  driver-os-monitor:
     build:
-        context: ./databox-os-monitor-driver
+        context: ./driver-os-monitor
         dockerfile: Dockerfile${DATABOX_ARCH}
```
Comment (mor1) [inline docker-compose-dev-local-images.yaml:9]: Only some of these clauses use `Dockerfile${DATABOX_ARCH}`, not all?
Also wouldn't it be clearer to use something like `Dockerfile.${DATABOX_ARCH}`, ie., include a separator?

**Review by mor1 — APPROVED** [review]
Left a couple of comments. Looks basically fine for this set of changes to me.

File: docker-compose-dev-local-images.yaml (hunk 3 above)
Comment (Toshbrown) [reply docker-compose-dev-local-images.yaml:9]: Fixed in 26f72ba2b4dc436b104ed4cb307dde959ac43886

On x86 DATABOX_ARCH == "" so it points to the default Dockerfile adding a separator here breaks this.

File: .gitignore (hunk 2 above)
Comment (Toshbrown) [reply .gitignore:88]: core-exporter is a bit vague, let's stick with core-export-service

**Review by ktg — APPROVED** [review]
Appears to run without issue

File: docker-compose-dev-local-images.yaml (hunk 3 above)
Comment (mor1) [reply docker-compose-dev-local-images.yaml:9]: My personal preference would be to make it explicit on all architectures. More consistent to maintain going forwards. (Avoid special cases!)

**Pull request merged**